style.css

```diff
@@ -57,7 +57,7 @@ header {
   width: 100%;
   background-color: transparent;
   border: 2px solid var(--primary-color);
-  border-radius: 50px;
+  border-radius: 5px;
```

Comment:
Hello, thanks for the contribution. However, I think it looks better with 50px 😄